app/helpers/menu_helper.rb

```diff
@@ -101,6 +101,6 @@ def settings_link
   end
 
   def current_api
-    (@backend_api || @service || Service.none).decorate
```

Comment:
isn't `Service.none` there for a reason?

Reply:
it's supposed to return `nil` when there is no service. `Service.None` returns an empty array and mess things up. AFAIR `current_api` is used nowhere else.

Reply:
But given it's a helper method it can be called from any view so I think it should be robust enough to deal with a no service situation. But probably input needed from ruby engineer.

Reply:
it might have been because `nil` raised from where `current_api` was called... I agree that an empty array is no way an indicator of the current api but I'd double check the usage to avoid panic.

Reply:
@thomasmaas a no service situation would be @nil, since this method is supposed to return 1 instance of BackendApi. For a method that returns a collection, `Service.none` would be appropriate.

I double checked it and it's only used by `vertical_nav`, so we're safe.